torchquad/integration/vegas.py

```diff
@@ -134,7 +140,7 @@ def integrate(
                 if chi2 / 5.0 < 1.0:
                     self._starting_N = torch.minimum(
                         torch.tensor(self._starting_N + self._N_increment),
-                        self._starting_N * torch.sqrt(acc / eps_rel),
+                        self._starting_N * torch.sqrt(acc / (eps_rel + 1e-8)),
```

Comment:
Isn't 1e-8 a bit high for an eps value?

Reply:
It is actually rather small.

See row binary32 machine epsilon [https://en.wikipedia.org/wiki/Machine_epsilon#Values_for_standard_hardware_floating_point_arithmetics](here).